src/lt/objs/find.cljs

```diff
@@ -221,13 +222,14 @@
               :desc "Editor: Go to line"
               :options line-input
               :exec (fn [l]
-                      (when (or (number? l) (not (empty? l)))
-                        (let [cur (pool/last-active)]
-                          (editor/move-cursor cur {:ch 0
-                                                   :line (dec (if-not (number? l)
-                                                                (js/parseInt l)
-                                                                l))})
-                          (editor/center-cursor cur))))})
+                      (let [line (if-not (number? l)
+                                   (js/parseInt l)
+                                   l)]
+                        (if (pos? line)
+                          (let [cur (pool/last-active)]
+                            (editor/move-cursor cur {:ch 0 :line (dec line)})
+                            (editor/center-cursor cur))
+                          (notifos/set-msg! (str "Line " l " is not a positive integer > 0") {:class "error"}))))})
```

Comment:
For clarity, perhaps change the message to be `"Line " l " must be a positive integer"`